src/elements/Options/Options.coffee

```diff
@@ -157,15 +157,33 @@ class CUI.Options extends CUI.DataField
 		@
 
 	disableOption: (value) ->
+		if @.__getOptionByValue(value).indeterminate then @removeIndeterminate(value)
 		cb = @__getCheckboxByValue(value)
 		cb.disable()
 		@
 
 	enableOption: (value) ->
+		if @.__getOptionByValue(value).indeterminate then @removeIndeterminate(value)
```

Comment:
Please move the function invocation below the if, remove the 'then'. (it's not wrong, it's just the way we usually do it)